ApollosProject/apollos-prototype
# Render Devotionals That Have No Scripture

## DESCRIPTION

### What does this PR do, or why is it needed?

Renders devotional content when there are no scripture references.

A bug was introduced with the addition of the experimental pager in TabView. This would cause any TabView with <2 tabs to fail to render any content. To fix this without reverting our previous fixes, we've added a condition that simply renders the actual tab content directly when there is only one tab.

Additionally, we added a loading state.

### What design trade-offs/decisions were made?

### How do I test this PR?

1. Open a devotional that has no scripture; it should render the devotional content
2. Open a devotional that DOES have scripture; it should render the tabview

### What automated tests did you write?

## SCREENSHOTS

|Scripture|No Scripture|
|---|---|
|<img width="570" alt="Screen Shot 2019-11-20 at 1 45 45 PM" src="https://user-images.githubusercontent.com/29125070/69267942-2f06cd00-0b9c-11ea-9c46-25eaa70170d8.png">|<img width="570" alt="Screen Shot 2019-11-20 at 1 45 26 PM" src="https://user-images.githubusercontent.com/29125070/69267951-33cb8100-0b9c-11ea-93c4-20e92b630bf7.png">|

## TODO:

- [x] Closes #1228 

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
